### PR TITLE
Add Contributing files

### PR DESCRIPTION
**Pull request by filariow**
Signed-off-by: Francesco Ilario <filario@redhat.com>

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+management@primaza.io .
```
Comment (baijum) [inline CODE_OF_CONDUCT.md:64]: Do you know if we have this email address configured?

Comment (sadlerap) [reply CODE_OF_CONDUCT.md:64]: As far as I know, no, we don't have emails registered yet for this domain.

Comment (filariow) [reply CODE_OF_CONDUCT.md:64]: that's a placeholder for now, we need to create a mail address for this

Comment (dperaza4dustbit) [reply CODE_OF_CONDUCT.md:64]: Lets leave placeholder and not require the email to be there, I don't want to trigger more product review protocols that are not needed now.

**Review by dperaza4dustbit — APPROVED** [review]

**Review by baijum — APPROVED** [review]

**Pull request merged**